src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -229,6 +233,31 @@ private function extractFromConstructor(string $class, string $property): ?array
         return null;
     }
 
+    private function extractFromDefaultValue(string $class, string $property)
+    {
+        try {
+            $reflectionClass = new \ReflectionClass($class);
+        } catch (\ReflectionException $e) {
+            return null;
+        }
+
+        $defaultValues = $reflectionClass->getDefaultProperties();
+
+        if (!isset($defaultValues[$property])) {
```

Comment:
@fabpot after deeping in this feature, I realized that all attributes has default values, even with `null` value. So no need to check for the existence of property here. It's ok to make this change?